.env.development

```diff
@@ -1,3 +1,4 @@
-VUE_APP_TITLE ="Find and Book parking spaces nearby | Bangalore Delhi Mumbai Pune Bengaluru | Parkspot.in"
+VUE_APP_TITLE="Find and Book parking spaces nearby | Bangalore Delhi Mumbai Pune Bengaluru | Parkspot.in"
 VUE_APP_MAP_ACCESS_TOKEN="pk.eyJ1IjoiYmZyaWVkbHkiLCJhIjoiY2p4bHd1OXdpMGFycDN0bzFiNWR4d2VyNyJ9.3hQjvgyoPoCuRx-Hqr_BFQ"
-NODE_ENV="development"
\ No newline at end of file
+NODE_ENV="development"
+VUE_APP_GOOGLE_MAP_TOKEN="AIzaSyAtHsC3zbjqgFkphcIjgG5OhrISlJ0bOaI"
```

Comment:
should this be coming from firebase.
Its a public repo any one can use our token. and if possible please discard this token and generate a new one because now its permanently visible.

Reply:
token changed and retrieve from firebase